master/reference/cni-plugin/configuration.md

```diff
@@ -432,3 +432,7 @@ take on the following precedence, 1 being the highest:
 > a newly configured IP pool, they must be recreated. We recommmend doing this
 > before going into production or during a maintenance window.
 {: .alert .alert-info}
+> **Note**: A CNI issue exists with the portmap plugin where draining nodes 
```

Comment:
I don't think this note belongs in the "Order of precedence" section.

We should probably add a new section called "CNI plugin chaining" which explains that we support chaining, and enable the portmap plugin by default, then include this note there.